projects/documentation/content/_includes/partials/sidenav.njk

```diff
@@ -130,7 +136,7 @@
                 href="{{ migration.url }}"
                 value="{{ migration.data.slug }}"
             >
-                <a href="{{ migration.url }}" slot="no-js">{{ migration.data.displayName }}</a>
+                {# <a href="{{ migration.url }}" slot="no-js">{{ migration.data.displayName }}</a> #}
```

Comment:
We need this to allow for the correct active item to be highlighted.

Reply:
This is buggy! I have reverted for now! We can work on this in a separate PR.